package.json

```diff
@@ -49,6 +50,9 @@
     "redux-thunk": "^2.1.0",
     "sass-loader": "^4.0.0",
     "style-loader": "^0.13.1",
+    "testcafe": "^1.8.1",
```

Comment:
It is not recommended to use ^ for packages.
You can get an updated package, but there is no any guarantee of compatibility -> the regression is possible.
In TMA we specify the versions statically and update them when upgrade is really required.